recipes/msaboot/meta.yaml

```diff
@@ -1,14 +1,14 @@
 package:
   name: msaboot
-  version: "0.1.1"
+  version: "0.1.2"
 
 source:
-  fn: msaboot-0.1.1.tar.gz
-  url: https://github.com/phac-nml/msaboot/archive/v0.1.1.tar.gz
-  sha256: 15c3aeeab425433026b5bfb8223e03c108c24d036abb79280ef83cdde262ee8c
+  fn: msaboot-0.1.2.tar.gz
+  url: https://github.com/phac-nml/msaboot/archive/v0.1.2.tar.gz
+  sha256: c541c5a6b27cbe0614ddf069eebc803f92bf8aa55bd2a752e66d76fbb9fd469d
 
 build:
-  number: 1
+  number: 2
```

Comment:
when the package version is being increased, the build number should be set to zero.